File: online_node_api/models/car.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class Car extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      // define association here
    }
  };
  Car.init({
    F_carID: {
      type:DataTypes.STRING,
      primaryKey:true,
      allowNull:false
    },
    F_carRegistration: DataTypes.STRING,
    F_carDetail: DataTypes.STRING,
    F_carStatus: DataTypes.INTEGER
  }, {
    sequelize,
    modelName: 'Car',
    tableName: 'sys_car',
    timestamps:false
  });
  return Car;
};

File: Flutter_Stock_Original/lib/models/car.dart
// ignore_for_file: non_constant_identifier_names

class Car {
  String F_carID;
  String F_carRegistration;
  String F_carDetail;
  int F_carStatus;

  Car(
      {required this.F_carID,
      required this.F_carRegistration,
      required this.F_carDetail,
      required this.F_carStatus});

  factory Car.fromJson(Map<String, dynamic> json) {
    return Car(
        F_carID: json['F_carID'],
        F_carRegistration: json['F_carRegistration'],
        F_carDetail: json['F_carDetail'],
        F_carStatus: json['F_carStatus']);
  }
}
